cmd/dcrtime/dcrtime.go

```diff
@@ -19,7 +19,7 @@ import (
 	"os"
 	"strconv"
 
-	v1 "github.com/decred/dcrtime/api/v1"
```

Comment:
We need the cli tool to support both versions. So you can't delete V1. It needs to become a config and cli flag.